angular/src/app/discovery/beacon/beacon-search-result-table/beacon-search-result-table.component.scss

```diff
@@ -3,5 +3,5 @@
 ddap-data-table {
   display: flex;
   flex-grow: 1;
-  overflow-y: scroll;
+  overflow: auto;
```

Comment:
Please stick with `overflow-y`. This change tells the browser to create a scrollbar for any axis if needed.